vladimir-vuckovic-devtech/laravel_learning
# Feature buttons create update delete

What's the purpose of this MR?

Purpose of this MR is to add buttons for creating, deleting and editing students in the database.

Impact Analysis

---

Where should the tester start?

Tester should start by going to /student url and check is there buttons for create, edit and delete.

How should this be manually tested?

Tester should click on each button and check do they address to right methods in StudentController.

Any background context you want to provide?

---

What are the relevant tickets?

Relevant ticket for this MR is "Add action buttons on the index page for create/update/delete".

Screenshots (if appropriate)

(N/A)
